.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
It's related to this change? It's not out of scope?

Reply:
Indeed it is out of scope. This file seems to be generated during `tox` run (or I messed something up). Will delete.